lib/index.js

```diff
@@ -83,10 +83,14 @@ PostHTML.prototype.process = function (tree, options) {
    */
   options = options || {}
 
-  if (options.parser) parser = options.parser
-  if (options.render) render = options.render
+  if (options.parser && typeof options.parser === 'function') {
+    parser = options.parser
+  }
+  if (options.render && typeof options.render === 'function') {
+    render = options.render
+  }
 
-  tree = options.skipParse ? tree : parser(tree)
+  tree = typeof tree === 'object' ? tree : parser(tree)
```

Comment:
hmm...it's not really 'breaking', ```options.skipParse``` just doesn't have any effect anymore :). Display warning when ```options.skipParse``` found?

Yep the initial  'error handling' messes up all test at the moment and there is no tree validation also, we can change/remove it if someone has a better solution.